src/sentry/static/sentry/app/views/performance/transactionSummary/latencyChart.tsx

```diff
@@ -71,8 +71,8 @@ class LatencyChart extends AsyncComponent<Props, State> {
       id: '',
       name: '',
       version: 2,
-      fields: [`histogram(transaction.duration,${NUM_BUCKETS})`, 'count()'],
-      orderby: 'histogram_transaction_duration_15',
+      fields: [`histogram_deprecated(transaction.duration,${NUM_BUCKETS})`, 'count()'],
```

Comment:
Getting this to use the same histogram with fencing like we use on measures would be 🤩

Reply:
That's the goal 👍 will be adding a new histogram end point that we'll be able to use to support all histograms going forwards.